drivers/storage/portworx/component/autopilot.go

```diff
@@ -45,7 +47,12 @@ const (
 	AutopilotDefaultProviderEndpoint = "http://px-prometheus:9090"
 	// AutopilotDefaultReviewersKey is a key for default reviewers array in gitops config map
 	AutopilotDefaultReviewersKey = "defaultReviewers"
-	defaultAutopilotCPU          = "0.1"
+	// OCPPrometheusUserWorkloadSecretPrefix name of OCP user-workload Prometheus secret
+	OCPPrometheusUserWorkloadSecretPrefix = "prometheus-user-workload-token"
+	// Autopilot Secret name for prometheus-user-workload-token
+	AutopilotSecretName                 = "autopilot-prometheus-auth"
+	defaultAutopilotCPU                 = "0.1"
+	OpenshiftPrometheusSupportedVersion = "4.14"
```

Comment:
Can we also move this constant to pxutil and use it in portworx.go and the tests instead of hard coding 4.14 there?